src/index.js

```diff
@@ -742,7 +742,11 @@ export function useDropzone(options = {}) {
         return;
       }
 
-      if (event.keyCode === 32 || event.keyCode === 13) {
+      if (
+        (event.key === 'Enter' || event.key === ' ') ||
+        (event.code === 'Enter' || event.code === 'Space') ||
```

Comment:
From MDN:
```
For example, the code returned is "KeyQ" for the Q key on a QWERTY layout keyboard, but the same code value also represents the ' key on Dvorak keyboards and the A key on AZERTY keyboards. That makes it impossible to use the value of code to determine what the name of the key is to users if they're not using an anticipated keyboard layout.
```

So `evt.code` is also not reliable. But `evt.key` seems like it is. So I don't think we need this one.

Reply:
Alright, just added it cause I wasn't sure